src/bin/driver.rs

```diff
@@ -119,6 +119,28 @@ impl Default for Settings {
     }
 }
 
+#[derive(Serialize, Copy, Clone, Default, Debug)]
+pub struct Monitor {
+    output_current: [f32; 2],
+    output_voltage: [f32; 2],
+    cpu_temp: f32,
+    overtemp: bool,
+}
+
+#[derive(Serialize, Copy, Clone, Default, Debug)]
+pub struct LowNoise {
```

Comment:
`LowNoiseControl{adc0, current}`?

Try sticking to the names on the schematic/panel.
If this is a current sink, is `output_current` really a useful name? What about just `current`? It's clear from the context that this is not the monitor data.

Reply:
What about the struct name?